.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
-*         @parfeon @CraigLn @kleewho @seba-aln
-.github/* @parfeon @CraigLn @kleewho @seba-aln
-README.md @techwritermat @kazydek
```

Comment:
non blocking comment: Talked with Karolina the other day and we agreed that for docs codeowners we'll have Her and Michal as a backup if @techwritermat is away.

Reply:
Returned back :)